SoomlaiOSProfile/ConfirmationDialog.h

```diff
@@ -4,7 +4,7 @@
 //
 
 #import <Foundation/Foundation.h>
-
+#import <UIKit/UIKit.h>
```

Comment:
@borisspektor, I didn't find why it worked earlier, but classes with UI prefix requires UIKit import.